sorc/chgres_cube.fd/surface.F90

```diff
@@ -2234,6 +2612,498 @@ subroutine calc_liq_soil_moisture
+    call error_handler("IN FieldGet", rc)
+
+ print*,"- CALL FieldGet FOR TARGET GRID ALNSF."
+ call ESMF_FieldGet(alnsf_target_grid, &
+                    farrayPtr=alnsf_target_ptr, rc=rc)
+ if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
+    call error_handler("IN FieldGet", rc)
+
+ print*,"- CALL FieldGet FOR TARGET GRID ALVWF."
+ call ESMF_FieldGet(alvwf_target_grid, &
+                    farrayPtr=alvwf_target_ptr, rc=rc)
+ if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
+    call error_handler("IN FieldGet", rc)
+
+ print*,"- CALL FieldGet FOR TARGET GRID ALNWF."
+ call ESMF_FieldGet(alnwf_target_grid, &
+                    farrayPtr=alnwf_target_ptr, rc=rc)
+ if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
+    call error_handler("IN FieldGet", rc)
+
+ print*,"- CALL FieldGet FOR TARGET GRID VEG FRAC."
+ call ESMF_FieldGet(veg_greenness_target_grid, &
+                    farrayPtr=veg_greenness_target_ptr, rc=rc)
+ if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
+    call error_handler("IN FieldGet", rc)
+
+ print*,"- CALL FieldGet FOR TARGET GRID MAX VEG FRAC."
+ call ESMF_FieldGet(max_veg_greenness_target_grid, &
+                    farrayPtr=max_veg_greenness_target_ptr, rc=rc)
+ if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
+    call error_handler("IN FieldGet", rc)
+
+ print*,"- CALL FieldGet FOR TARGET GRID MIN VEG FRAC."
+ call ESMF_FieldGet(min_veg_greenness_target_grid, &
+                    farrayPtr=min_veg_greenness_target_ptr, rc=rc)
+ if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
+    call error_handler("IN FieldGet", rc)
+
+ print*,"- CALL FieldGet FOR TARGET GRID CANOPY MC."
+ call ESMF_FieldGet(canopy_mc_target_grid, &
+                    farrayPtr=canopy_mc_target_ptr, rc=rc)
+ if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
+    call error_handler("IN FieldGet", rc)
+
+ print*,"- CALL FieldGet FOR TARGET GRID SNOW ALBEDO."
+ call ESMF_FieldGet(mxsno_albedo_target_grid, &
+                    farrayPtr=mxsno_albedo_target_ptr, rc=rc)
+ if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
+    call error_handler("IN FieldGet", rc)
+
+ do i =clb(1),cub(1)
+   do j = clb(2),cub(2)
+     if (landmask_ptr(i,j)==1 .and. soilm_target_ptr(i,j,1) > 0.75) then
+       !write(*,'(a,2i5,a,i3,f5.2,2i3)') "CORRECTING G.P.",i,j," FROM LAND TO SEA VALUES; &
+       !    curr landmask, soilm, stype, vtype = ",landmask_ptr(i,j),&
+       !    soilm_target_ptr(i,j,1),nint(soil_type_target_ptr(i,j)), &
+       !    nint(veg_type_target_ptr(i,j))
+       soil_type_target_ptr(i,j) = 0.0
+       veg_type_target_ptr(i,J) = 0.0
+       landmask_ptr(i,j) = 0
```

Comment:
I would not change the land mask in chgres_cube.  The coldstart files will not match the 'orog' files.

Reply:
@GeorgeGayno-NOAA This is old code that, due to other routines, never gets called anymore.  We will comment it out and issue a commit to this PR.

Reply:
If you'll look in the driver subroutine, the call to that subroutine is commented out. I'm going to comment out the entire subroutine for now. I'm reticent to remove it completely just in case we need it again.

Reply:
Why would you need it?  If your input file has a proper mask and good data, then you should be able to map to any target/model grid.  You should never have to change the target land mask.

Reply:
It was a quick fix that was implemented to a problem we were having with soil moisture mismatches just to see if we could fix the problem. We recognized that this sort of solution was not appropriate and implemented other more appropriate fixes elsewhere in the code. Again, this code is no longer used.